src/main/java/in/org/projecteka/hiu/dataflow/model/DataEntry.java

```diff
@@ -12,6 +13,6 @@
 public class DataEntry {
     private String hipId;
     private String hipName;
-    private Status status;
+    private EntryStatus status;
```

Comment:
For me status made more sense than having redundant prefix Entry, as it was inside DataEntry object already, it was supposed to represent DataEntry status only.

Reply:
there are 2 different status fields .. we wanted to keep the distinction.